api/src/main/java/io/minio/ObjectStat.java

```diff
@@ -29,22 +29,30 @@
   private final long length;
   private final String etag;
   private final String contentType;
-
+  private final String key;
+  private final String iv;
+  private final String matDesc;
```

Comment:
- x-amz-iv	Randomly generated IV (per S3 object), base64 encoded
- x-amz-matdesc	Customer provided material description in JSON (UTF8) format. Used to identify the client-side master key (ie used to encrypt/wrap the generated content encrypting key).